server/integration/testsuite/src/test/java/org/infinispan/persistence/cluster/MyCustomCacheStore.java

```diff
@@ -8,6 +8,10 @@
 import org.infinispan.util.logging.Log;
 import org.infinispan.util.logging.LogFactory;
 
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.fail;
```

Comment:
AFAIK TestNG and JUnit both throw AssertionError, the only difference is the parameter order for `assertEquals`. Our convention so far is to use `AssertJUnit` in TestNG tests, because its `assertEquals` parameter order mimics JUnit's, and we want to migrate to JUnit at some point. But I don't think there's any need to use `AssertJUnit` instead of JUnit's `Assert`.